Cannot find module 'win-ca/fallback'

&lt;Please be sure to remove any private information before submitting.&gt;

Repro steps:
&lt;Enter steps to reproduce issue&gt;

Action: docker.certificates
Error type: MODULE_NOT_FOUND
Error Message: Cannot find module 'win-ca/fallback'

Version: 0.5.2
OS: win32

--- a/package.json
+++ b/package.json
@@ -952,7 +952,7 @@
     "tar": "^4.4.6",
     "vscode-azureextensionui": "^0.19.0",
     "vscode-languageclient": "^5.1.1",
-    "win-ca": "^2.2.0",
+    "win-ca": "^2.4.0",
     "xml2js": "^0.4.19"
   }
 }

